app-crypt/princeprocessor/princeprocessor-0.22-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit edo toolchain-funcs
+
+DESCRIPTION="Standalone password candidate generator using the PRINCE algorithm"
+HOMEPAGE="https://github.com/hashcat/princeprocessor"
+SRC_URI="https://github.com/hashcat/princeprocessor/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P}/src"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_compile() {
+	edo $(tc-getCC) -W -Wall -std=c99 ${CFLAGS} ${LDFLAGS} -m64 -DLINUX -o ${PN} pp.c mpz_int128.h
+}
+
+src_install() {
+	dobin ${PN}
+	dodoc ../{README.md,CHANGES}
+	#install rules after hashcat is fixed
```

Comment:
What about those comments? Are they needed?

Reply:
Actually i don't know. 
I was looking for some bug / more information about that but couldn't find anything.
Maybe @ZeroChaos- can comment on this?